zebra/kernel_netlink.c

```diff
@@ -406,7 +406,6 @@ static int netlink_information_fetch(struct nlmsghdr *h, ns_id_t ns_id,
 	case RTM_DELROUTE:
 		return netlink_route_change(h, ns_id, startup);
 	case RTM_NEWLINK:
-		return netlink_link_change(h, ns_id, startup);
```

Comment:
we've got a block a little further down where the dplane messages are kept together - it'd be more consistent to put these LINK messages there too?

Reply:
Why/how is this useless?

Reply:
Done.

Reply:
> we've got a block a little further down where the dplane messages are kept together - it'd be more consistent to put these LINK messages there too?

Done. Thanks for your review.